src/GNNGraphs/transform.jl

```diff
@@ -149,6 +149,51 @@ function remove_self_loops(g::GNNGraph{<:ADJMAT_T})
              g.ndata, g.edata, g.gdata)
 end
 
+"""
+    remove_edges(g::GNNGraph, edges_to_remove::Vector{Int})
```

Comment:
```suggestion
    remove_edges(g::GNNGraph, edges_to_remove::AbstractVector{<:Integer})
```
and the signature should be updated